src/order_tree.cpp

```diff
@@ -0,0 +1,204 @@
+#pragma once
+#include "C:\Users\Hamza Rashid\Desktop\C++\Trading Engine\include\TradingEngine\order_tree.h"
+
+#include <cmath>
+#include <memory>
+using std::shared_ptr;
+
+#include "C:\Users\Hamza Rashid\Desktop\C++\Trading Engine\include\TradingEngine\order.h"
+#include "C:\Users\Hamza Rashid\Desktop\C++\Trading Engine\include\TradingEngine\transactions.h" 
+#include "C:\Users\Hamza Rashid\Desktop\C++\Trading Engine\include\TradingEngine\limit_order.h"
+
+/* Return length of Order's within the Tree */
+int OrderTree::get_length() const {
+	return order_map_.size();
+}
+
+/* Get min price within Ordertree */
+float OrderTree::get_min() {
```

Comment:
The getters should have the same exact name as the variable, so here it should be `getMinPrice`